__test__/backend.test.js

```diff
@@ -805,3 +824,185 @@ describe("editUser mutation", () => {
     });
   });
 });
+
+describe("A tag table", () => {
```

Comment:
please move these to a new file -- we don't add anything to backend.test.js anymore.  how about `__test__/server/models/tag.js`? or server/tag.js, especially if you plan on populating more later with some more application-y code.

Reply:
Done!